lms/djangoapps/instructor_task/rest_api/v1/urls.py

```diff
@@ -0,0 +1,24 @@
+"""
+Instructor Task Django app REST API URLs.
+"""
+from django.conf import settings
+from django.urls import re_path
+
+from lms.djangoapps.instructor_task.rest_api.v1.views import (
+    ListScheduledBulkEmailInstructorTasks,
+    DeleteScheduledBulkEmailInstructorTask
+)
+
+
+urlpatterns = [
+    re_path(
+        fr"schedules/{settings.COURSE_ID_PATTERN}/bulk_email/$",
```

Comment:
This results in a path that looks something like... `/api/instructor_task/schedules/course-v1:blah+blah101+1T2022/bulk_email/`